functions/Install-DbaMaintenanceSolution.ps1

```diff
@@ -53,6 +53,9 @@ function Install-DbaMaintenanceSolution {
     .PARAMETER Confirm
         If this switch is enabled, you will be prompted for confirmation before executing any operations that change state.
 
+    .PARAMETER InstallParallel
+        If this switch is enabled, the Queue and QueueDatabase tables are created.
```

Comment:
Could you add a note that these tables are used by the DatabasesInParallel option?

Reply:
That has been done, waiting to push/commit for any further tweaks.